genesis/make_genesis_file.go

```diff
@@ -28,6 +28,25 @@ func GenerateKnown(chainID, accountsPathCSV, validatorsPathCSV string) (string,
 }
 
 //------------------------------------------------------------------------------------
+// interface functions that are consumed by monax tooling
+
+func GenerateGenesisFileBytes(chainName string, genesisAccounts []*GenesisAccount,
```

Comment:
in tooling there is a usage of passing slices of pointers; which is justified there because they are passed around a lot during construction;

this is why i deliberately gave tooling an interface that respects the `[ ]*T`; but `GenesisDoc` is constructed with `[ ]T` and once `GenesisDoc` is constructed changing the original `T` should not affect the `GenesisDoc`